### DISCUSSION TITLE
Memory widget ?

### DISCUSSION THREAD

**Discussion by MTVSilentservic** [seerge/g-helper#3428] [General]
I was wondering if it would be possible to have a Memory and storage tab like in AC ? I like the simplicity of being able to glance over to see how much storage I have remaining overall.

**Comment by seerge** [comment]
@MTVSilentservic hello, memory widget in AC has an option to “free up” memory. I personally don’t see much sense in it as modern windows manages memory quite effectively, and it frees it up by itself only when it’s needed. So it’s better to let system do its job without interfering:)

**Reply by MTVSilentservic** [reply]
@seerge That makes since! I am new to gaming laptops and PCs. Thanks for the information! And quick response. You do awesome work out here!

**Reply by MTVSilentservic** [reply]
@seerge also when I hit my AC button on my computer to get g-helper to pop up, I keep getting an AC pop up message to install AC. I uninstalled AC using the uninstall tool. And way to keep this pop up from happening?

**Reply by seerge** [reply]
@MTVSilentservic stop Asus services from Extra settings in G-Helper

**Reply by MTVSilentservic** [reply]
@seerge i did that initially but my computer has updated and I haven't checked it again. Thank you!
